charts/rucio-server/values.yaml

```diff
@@ -166,6 +166,10 @@ policyPackages:
     # - moduleName: vo_2_policy_package
     #   requirement: git+https://github.com/vo-2/vo-2-policy-package@v0.1.0
     #   version: 0.1.0
+  # Set true to create a PVC for the policy packages; leave false if providing it separately.
+  createPvc: false
+  # If providing the PVC separately, set its name here.
```

Comment:
I am confused by this comment. ‘If separate, then do this.’ But when not ‘if separate’, don’t you need to set it to something that will be unique to the release?

Reply:
I see how the comment could be misleading. If not separate (i.e. `createPvc: true`), you'd still set this name to something unique to the release - the comment was mostly to indicate what to do in case you wanted to provide the PVC separately

Reply:
I've removed the comment, and refactored it a bit. I've added a bool to toggle whether the release name should be prepended to the PVC name. Example run:

```
helm template test-release charts/rucio-server -f charts/rucio-server/values.yaml --debug 
```

```
...
      - name: policy-package-volume
        persistentVolumeClaim:
          claimName: test-release-policy-package-volume
...
```